src/go/rpk/pkg/cli/cmd/wasm/remove.go

```diff
@@ -0,0 +1,64 @@
+package transform
+
+import (
+	"encoding/binary"
+
+	"github.com/Shopify/sarama"
+	"github.com/spf13/cobra"
+)
+
+func NewRemoveCommand(
+	createProduce func(bool, int32) (sarama.SyncProducer, error),
+) *cobra.Command {
+	var (
+		jvmPartitioner	bool
+		partition	int32
+	)
+
+	command := &cobra.Command{
+		Use:	"remove <name>",
+		Short:	"remove inline WASM function",
+		Args:	cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			name := args[0]
+			producer, err := createProduce(jvmPartitioner, partition)
+			if err != nil {
+				return err
+			}
+			return remove(
+				name,
+				producer,
+			)
+		},
+	}
+
+	command.Flags().BoolVarP(
+		&jvmPartitioner,
+		"jvm-partitioner",
+		"j",
+		false,
+		"Use a JVM-compatible partitioner. If --partition is passed"+
+			" with a positive value, this will be overridden and"+
+			" a manual partitioner will be used.",
+	)
+
+	command.Flags().Int32VarP(
+		&partition,
+		"partition",
+		"p",
+		-1,
+		"Partition to produce to.",
+	)
+
+	return command
+}
+
+func remove(name string, producer sarama.SyncProducer) error {
+	// create buffer for filename and content size bytes
+	sizeName := make([]byte, 16)
+	binary.LittleEndian.PutUint16(sizeName, uint16(len([]byte(name))))
+	// join all bites on one buffer
+	messageContent := append(sizeName, []byte(name)...)
```

Comment:
If we do agree we want to add this type of metadata to our coprocTopic messages, we should make a helper method
```
func createCoprocTopicMessage(name, description, content) error {
   ....
}
```
and place it in a place remove.go and deploy.go can both access it

Reply:
Umm, I can move those process to different functions, but I can't use the same for those commands, it uses different information

Reply:
solved in slack, we create a new function for creating deploy message but no remove message because that message doesn't need a message for working just key